### PR TITLE
core-1138 - Path list viewer - 2/n

### PR DESCRIPTION
**Pull request by sri2k1us**
- Create new Path List file (HT or Multi input)
- Save newly created file
- Edit existing Path List file

image

- iPhone 5 
image

-iPhone 5 toolbar
image

- Create menu items
image

- Save as dialog
image

- Known Issue
    Users can leave the page without saving the edited path lists by hitting browser back button or in app navigations links. There will be no warnings or confirmation.

### CODE REVIEW

**Review by psarando — APPROVED** [review]
LGTM 🎉 
I only found a few nits to pick (and maybe 1 minor bug), but :shipit: when you feel it's ready! 👍

File: src/pages/data/ds/[...pathItems].js [hunk 1]
```diff
@@ -28,43 +31,93 @@ const dynamicPathName = "/[...pathItems]";
...
+                    `${baseRoutingPath}${encodedPath}`
+                );
+            } else {
+                router.push(
+                    `${baseRoutingPath}${dynamicPathName}?file=true&resourceId=${id}`,
+                    `${baseRoutingPath}${encodedPath}?file=true&resourceId=${id}`
+                );
+            }
+        },
+        [baseRoutingPath, router]
```
Comment (ash3rz) [inline src/pages/data/ds/[...pathItems].js:61]: 👍

**Review by ash3rz — APPROVED** [review]
LGTM :shipit: 🎉

**Comment by sri2k1us** [comment]
Thanks for the review.

**Pull request merged**